miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1609,21 +1609,32 @@ public static LibraryStrategyTypeDto asDto(LibraryStrategyType from) {
   }
 
   public static IndexDto asDto(Index from) {
+    return asDto(from, true);
+  }
+
+  public static IndexDto asDto(Index from, boolean includeFamily) {
     IndexDto dto = new IndexDto();
     dto.setId(from.getId());
     dto.setLabel(from.getLabel());
     dto.setName(from.getName());
     dto.setPosition(from.getPosition());
     dto.setSequence(from.getSequence());
+    if (includeFamily) {
+      dto.setFamily(asDto(from.getFamily(), false));
+    }
     return dto;
   }
 
   public static IndexFamilyDto asDto(IndexFamily from) {
+    return asDto(from, true);
+  }
+
+  private static IndexFamilyDto asDto(IndexFamily from, boolean includeChildren) {
```

Comment:
Yes. In the `asDto(IndexFamily)` it's true, while in the `asDto(Index)`, it's false.